sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/ProxyInfoHolder.scala

```diff
@@ -63,10 +63,10 @@ class ProxyInfoHolder(system: ExtendedActorSystem) extends Extension {
     _identificationInfo = proxyInfo.identificationInfo
     _proxyTracingCollectorEndpoint = Some(proxyInfo.tracingCollectorEndpoint)
 
-    log.debug("Proxy hostname: [{}]", chosenProxyName)
-    log.debug("Proxy port to: [{}]", proxyInfo.proxyPort)
```

Comment:
The protobuf type is still called `ProxyInfo`, but I think we can safely rename it in the runtime to `RuntimeInfo` and it will work anyway, no?

@johanandren could you confirm this? 
If I have two messages: `Foo` and `Bar` and they have the same fields in the same order, doesn't matter the name of the messages in the descriptor, we will always be able to deserialize a payload from `Foo` into `Bar`, correct?